src/webgpu/api/validation/texture/rg11b10ufloat_renderable.spec.ts

```diff
@@ -16,10 +16,20 @@ sampleCount > 1, iff rg11b10ufloat-renderable feature is enabled.
 Note, the createTexture tests cover these validation cases where this feature is not enabled.
 `
   )
-  .params(u =>
-    u.combine('usage', [0, GPUConst.TextureUsage.RENDER_ATTACHMENT]).combine('sampleCount', [1, 4])
-  )
-  .unimplemented();
+  .params(u => u.combine('sampleCount', [1, 4]))
```

Comment:
I noticed that `usage` must not be zero and `sampleCount` > 1 requires `RENDER_ATTACHMENT` usage so removed `0` usage test case because it's useless here. (I think `0` usage fail test should be done somewhere else.)

https://gpuweb.github.io/gpuweb/#abstract-opdef-validating-gputexturedescriptor

Reply:
I checked to make sure we didn't have missing coverage. It looks like 0 usage is already covered:
https://github.com/gpuweb/cts/pull/1575
Not sure about sampleCount without RENDER_ATTACHMENT, so I added the `needs-cts-issue` label to
https://github.com/gpuweb/gpuweb/pull/2786
to make sure we cover it.

Reply:
This test case seems to cover `sampleCount > 1` without `RENDER_ATTACHMENT` usage. https://github.com/gpuweb/cts/blob/9c96bdc381db9aee24a445eac15f577d012fd91a/src/webgpu/api/validation/createTexture.spec.ts#L309-L390